### PR TITLE
create, update and delete function created

### PR DESCRIPTION
**Pull request by Tavneet1430**
3 Functions are done

### CODE REVIEW

File: src/main/java/com/softech/coursesvc/controller/CourseController.java [hunk 1]
```diff
@@ -38,16 +35,29 @@ public CourseList getCourses() {
     }
 
 @GetMapping("/v1/course/{courseId}")
-public ResponseEntity<CourseEntity> getCourseById(@PathVariable Long courseId)
-{
-    CourseEntity course= courseSvc.getCourseById(courseId);
-    if(course != null)
-    {
+public ResponseEntity<CourseEntity> getCourseById(@PathVariable Long courseId) {
+    CourseEntity course = courseSvc.getCourseById(courseId);
```
Comment (richa002) [inline src/main/java/com/softech/coursesvc/controller/CourseController.java:39]: Are you working on get by id endpoint? I guess neetu worked on it..why you have this added

Comment (richa002) [reply src/main/java/com/softech/coursesvc/controller/CourseController.java:39]: Line 38 39, if you added them , remove them

Comment (heenakh30) [reply src/main/java/com/softech/coursesvc/controller/CourseController.java:39]: She had to work on create, update and delete endpoint

Comment (Tavneet1430) [reply src/main/java/com/softech/coursesvc/controller/CourseController.java:39]: When I clone the project, it automatically appears in my project list

**Review by richa002 — APPROVED** [review]

File: src/main/java/com/softech/coursesvc/service/CourseSvc.java [hunk 2]
```diff
@@ -21,4 +21,19 @@ public List<CourseEntity> getAllCourses() {
     public CourseEntity getCourseById(Long courseId) {
         return courseRepos.findById(courseId).orElse(null);
     }
+
+     public CourseEntity createCourse(CourseEntity courseEntity) {
+
```
Comment (richa002) [inline src/main/java/com/softech/coursesvc/service/CourseSvc.java:26]: remove extra lines

**Pull request merged**